pkgs/development/tools/vala-language-server/default.nix

```
@@ -0,0 +1,46 @@
+{ stdenv
+, fetchFromGitHub
+, meson
+, ninja
+, pkg-config
+, scdoc
+, glib
+, libgee
+, json-glib
+, jsonrpc-glib
+, vala
+}:
+
+stdenv.mkDerivation rec {
+  pname = "vala-language-server";
+  version = "0.48.1";
+  src = fetchFromGitHub {
```

can you add whitespace here 
```suggestion

  src = fetchFromGitHub {
```